scripts/generate-hsluv-tests.js

```diff
@@ -0,0 +1,115 @@
+import fs from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import https from "node:https";
+
+const testDataUrl = "https://raw.githubusercontent.com/hsluv/hsluv/ebe5f6fbe7082e04468a325b4c62f783e35987ad/snapshots/snapshot-rev4.json";
```

Comment:
Cool idea to import the tests from that repo!
However, if we're gonna pin it to a specific commit, we may as well just copy the file over and `import` it in the test. There is however value in loading the latest version of the file. Not sure what route you'd rather go with.

Reply:
The [test data hasn't changed in 8 years](https://github.com/hsluv/hsluv/tree/master/snapshots) so I guess there's no value in pinning to a specific commit. I don't think the file will ever change and if changes are required a `snapshot-rev5.json` will be created.

My preference right now would be to do the least amount of work. So either keep it as is or change the `testDataUrl` to not reference a specific commit.

Reply:
Walked the dog and changed my mind.

- I'll create a `test/hsluv` directory that contains a copy of the json test data and test files for hpluv and hsluv
- the test files will load the json test data and create the appropriate tests
- the `test:generate-hsluv` script will be deleted
- `npm run test:hsluv` will run the hsluv tests